app/models/beta/search/intercept_message.rb

```diff
@@ -0,0 +1,22 @@
+module Beta
+  module Search
+    class InterceptMessage
+      include ContentAddressableId
+
+      content_addressable_fields :intercept_message
```

Comment:
```suggestion
      content_addressable_fields :term, :message
```

Reply:
We need the :intercept_message field here otherwise we get this error from the concern 
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/108667911/182802368-03b9a3d4-efa6-4244-b78e-7476dd5e1403.png">

Reply:
:eyes: